daemon/daemon.go

```diff
@@ -612,10 +612,9 @@ func (d *daemon) RestartTeamLab(req *pb.RestartTeamLabRequest, resp pb.Daemon_Re
 		return err
 	}
 
-	lab, err := ev.GetHub().GetLabByTag(req.LabTag)
-
-	if err != nil {
-		return err
+ 	lab,ok:=ev.GetLabByTeam(req.TeamId);
```

Comment:
Please make sure that the code is either formatted by `go fmt` or `goimports`, most IDEs should do this on default as you save go files.

Reply:
👍🏻